consistency/tool/clustering/word_extraction.py

```diff
@@ -7,32 +7,45 @@ def __init__(self, stub):
       self.stub = stub
 
     def extract_vocabs(self):
-      stub = self.stub
-      try:
-          response = stub.ListArtifacts(
-              registry_service_pb2.ListArtifactsRequest(
-                  parent="projects/-/locations/global/apis/-/versions/-/specs/-",
-                  filter="name.contains(\"vocabulary\")"
-              ))
-      except grpc.RpcError as rpc_error:
-          print(f"Received RPC error: code={rpc_error.code()} message={rpc_error.details()}") 
-
-      vocabs = []
-     
-      for artifact in response.artifacts:
-          contents = stub.GetArtifactContents(
-              registry_service_pb2.GetArtifactContentsRequest(
-                  name=artifact.name,
-              )
-          )
-          vocab = vocabulary_pb2.Vocabulary()
-          vocabs.append(vocab.ParseFromString(contents.data))
-
-      return vocabs
+        stub = self.stub
+        try:
+            response = stub.ListArtifacts(
+                registry_service_pb2.ListArtifactsRequest(
+                    parent="projects/-/locations/global/apis/-/versions/-/specs/-",
+                    filter="name.contains(\"vocabulary\")"
+                ))
+        except grpc.RpcError as rpc_error:
+            print(f"Received RPC error: code={rpc_error.code()} message={rpc_error.details()}") 
```

Comment:
Can we make this error message more descriptive like "Failed to fetch vocabulary artifacts, RPC error: ..."